ledger/catchpointwriter.go

```diff
@@ -109,7 +107,6 @@ func makeCatchpointWriter(ctx context.Context, filePath string, tx *sql.Tx) (*ca
 		totalAccounts: totalAccounts,
 		totalChunks:   (totalAccounts + BalancesPerCatchpointFileChunk - 1) / BalancesPerCatchpointFileChunk,
 		file:          file,
-		gzip:          gzip,
```

Comment:
I think we should keep compression for the data file. Don't know if 10-15x IO is alright.

Reply:
wow, that's the size difference between compressed and uncompressed catchpoints? I had also considered using something cheap like snappy, or even gzip with the `gzip.BestSpeed` level